src/database/comment.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose from 'mongoose';
+import IComments from '../interfaces/comment';
+
+const commentsSchema = new mongoose.Schema<IComments>({
+  commentBody: {
+    type: String,
+  },
+  user: {
+    type: String,
+    ref: 'users',
+  },
+  announcement: {
```

Comment:
change announcement to announcementId because its also a ref to another collection